src/search_count_report.py

```diff
@@ -0,0 +1,46 @@
+import csv
+from datetime import datetime
+
+from src.report import Report
+import src.utils as utils
+
+class SearchCountReport(Report):
+
+  def generate(self):
+    # Write fields to search_count.csv 
+    self.writer.writerow(['day', 'hour', 'count'])
+
+    time_to_count_dict = {}
+
+    # Read routes.csv 
+    for index, row in enumerate(self.reader):
+      if index == 0:
+        continue
+      timestamp = row[0]
+      datetime = self.parse_date(timestamp)
+      hour = datetime.hour
+      day = self.parse_day(timestamp)
+      if day in time_to_count_dict:
+        if hour in time_to_count_dict[day]:
+          time_to_count_dict[day][hour] += 1
+        else:
+          time_to_count_dict[day][hour] = 1
+      else:
+        time_to_count_dict[day] = {hour: 1}
+
+    for day in time_to_count_dict:
+      for hour in time_to_count_dict[day]:
+        self.writer.writerow([day, hour, time_to_count_dict[day][hour]])
+
+  def parse_date(self, timestamp):
+    suffixes = ['st', 'th', 'rd']
+    time_str = timestamp
+    # Strip off ordinal suffixes so we can easily convert to datetime object
+    for s in suffixes:
+      time_str = time_str.replace(s, '')
+    # Strip the milliseconds off of the time
+    time_str = time_str.split('.')[0]
+    return datetime.strptime(time_str, "%B %d %Y, %H:%M:%S")
+
+  def parse_day(self, timestamp):
```

Comment:
`parse_day_str_from_timestamp`? or `parse_day_str_from_str`? or too wordy lmk

Reply:
i think parse_day is clear enough